files/scripts/startup_tsa_tsb.py

```diff
@@ -69,16 +69,33 @@ def config_tsa():
     if tsa_ena == True:
         logger.log_info("Configuring TSA")
         subprocess.check_output(['TSA']).strip()
+        logger.log_info("Setting TSA-TSB service field in CONFIG_DB")
```

Comment:
Do you want to move this to STATE_DB instead since it is state and not config

Reply:
yes.. Moved to STATE_DB..